src/server/letter-lists.js

```diff
@@ -0,0 +1,42 @@
+const consonants = {
```

Comment:
In the README it says v5 or above so this is fine with me too - just be careful for what is supported and isn't for v5 or update the README to require 6+.